metadata.rb

```diff
@@ -4,7 +4,7 @@
 license          "Apache 2.0"
 description      "Installs/Configures Sensu"
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          "2.12.1"
+version          "2.13.0"
```

Comment:
Please don't bump the version in this PR; we'll take care of it when we're ready to cut the next release.